### ISSUE TITLE
Status Code is not shown

### ISSUE THREAD

**Issue by jubakong** [kean/Pulse#58]
Pulse looks like it is such an amazing network logger, but `PulseUI` does not show `status code`.
I am using `Alamofire` for networking.

This red circle's supposed to be `fail` because the network response code returns `401`.
However, it is shown as `Success`.
![IMG_0397](https://user-images.githubusercontent.com/52398126/141399348-abd8ec88-141b-47ea-9507-5c783700e0e4.png)

This image is a detailed page of `above image` which returns `status code` as `401`. 
As image shown, `Status Code` displays as `-`.
![2](https://user-images.githubusercontent.com/52398126/141399703-8b3c9086-a981-407e-946b-52c0747c42b4.png)

Is there any parts that I am missing?
Thank you.

**Comment by kean** [comment]
Looks like a defect. Can you share how you logs the network requests, please?

**Comment by jubakong** [comment]
I am using `Alamofire` + `RxSwift` for getting return value as `Single`

```
  func codableDataRequest<T: Codable>(encoding: ParameterEncoding = JSONEncoding.default) -> Single<T> {
    return Single<T>.create { observer in
      let request = manager.request(
        endPoint,
        method: method,
        parameters: self.parameters,
        encoding: encoding,
        headers: headerAuthorization
      )
      
      request
        .handleCodable { (response: CodableResponseContainer<T>) in
          guard let isReachable = NetworkReachabilityManager()?.isReachable,
                isReachable
          else {
            let error = NSError(
              domain: "NETWORK_REACHABILITY_DOMAIN",
              code: -99,
              message: "Network is invalid"
            )
            return observer(.failure(error))
          }
          
          if let data = response.data {
            observer(.success(data))
          }
        } failure: { error in
          observer(.failure(error))
        }
      return Disposables.create {
        request.cancel()
      }
    }
  }

**Comment by kean** [comment]
Looks like this issue is specific to the `Alamofire.EventMonitor`. Unlike `Pulse.URLSessionProxyDelegate`, it doesn't have the following callback that provides a URLResponse:

```swift
public func urlSession(_ session: URLSession, dataTask: URLSessionDataTask, didReceive response: URLResponse, completionHandler: @escaping (URLSession.ResponseDisposition) -> Void) {
    ...    
}
```

I missed it when I wrote an example of `EventMonitor`.

Fortunately, it's not required. I'm going to ship a fix in today's release.

**Comment by kean** [comment]
Fixed in [0.20.0](https://github.com/kean/Pulse/releases/tag/0.20.0)

**Comment by jubakong** [comment]
Now, it's working great!
